packages/adblocker/src/engine/engine.ts

```diff
@@ -7,6 +7,7 @@
  */
 
 import type { IMessageFromBackground } from '@ghostery/adblocker-content';
+import { URLSearchParams } from '@ghostery/url-parser';
```

Comment:
Why not use the native implementation? As far as I know it's available in all environments including Node.js.

Reply:
I think so far we didn't use `URL` because there was no need. We only needed domain parsing, for which we used `tldts`. I'm not necessarily against using the `@ghostery/url-parser`, just curious why that was. Also, how much does it add to the minified bundle?

Reply:
With `url-parser` the minified bundle is `120.6 KiB` and `118.8 KiB` without.

Reply:
Sounds reasonable